src/components/errorBoundary/index.js

```diff
@@ -25,36 +22,30 @@ class ErrorBoundary extends React.Component {
   }
 
   render() {
+    const { t } = this.props;
     const getMailReference = () => {
       const recipient = 'hubdev@lisk.io';
       const subject = `User Reported Error - Lisk Hub - ${VERSION}`; // eslint-disable-line no-undef
       const body = `${this.state.error}:%0A${this.state.info.componentStack.replace(/\s{4}/g, '%0A')}`;
       return `mailto:${recipient}?&subject=${subject}&body=${body}`;
     };
-    const emptyStateProps = {
-      title: this.props.errorMessage,
-      message: this.props.t('To recover you can'),
-      className: styles.section,
-    };
+
     const renderErrorSection = () => (
-      <Box>
-        <EmptyState {...emptyStateProps}>
-          <div>
-            <PrimaryButton
-              theme={styles}
-              label={this.props.t('Reload the page')}
-              className='error-reload-btn'
-              onClick={() => this.reloadPage() }/>
-            <p className={styles.description}>{this.props.t('if the problem persists')}</p>
-            <a target='_blank'
-              className={styles.link}
-              href={getMailReference()}
-              rel='noopener noreferrer'>
-              {this.props.t('Report the error via E-Mail')}&nbsp;<FontIcon>arrow-right</FontIcon>
+        <section className={styles.errorBoundaryPage}>
+          <div className={styles.errorMessageContainer}>
+            <Illustration name={'errorBoundaryPage'}/>
+            <h2>{t('An error occurred.')}</h2>
+            <p>{t('To recover, you can try to reload the page, by clicking the button below. If the problem persists, report the error via email.')}</p>
+            <PrimaryButtonV2 className={`${styles.reloadPageButton} error-reload-btn`} onClick={this.reloadPage()}>
```

Comment:
```suggestion
            <PrimaryButtonV2 className={`${styles.reloadPageButton} error-reload-btn`} onClick={this.reloadPage}>
```
Otherwise it reloads the page on render

Reply:
ouch